models/migrations/v78.go

```diff
@@ -34,8 +34,17 @@ func renameRepoIsBareToIsEmpty(x *xorm.Engine) error {
 	} else if models.DbCfg.Type == core.MSSQL {
 		_, err = sess.Exec("DROP INDEX IF EXISTS IDX_repository_is_bare ON repository")
 	} else {
-		_, err = sess.Exec("DROP INDEX IDX_repository_is_bare ON repository")
+
+		indexes, err := sess.QueryString(`SHOW INDEX FROM repository WHERE KEY_NAME = 'IDX_repository_is_bare'`)
```

Comment:
I don't think this will work for sqlite

Reply:
Take a look at https://github.com/go-gitea/gitea/pull/6736/commits/9715cac7b6fe2cf57dbd4b084db12d82284bed71